### PR TITLE
Fixed typo in Dockerfile

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by kshakir** [comment]
Tested locally. Wheel chose @scottfrazer and @geoffjentry. cc'ing @coreone

**Comment by geoffjentry** [comment]
Ridiculously TOL (and I even hate the phrase) - this reminds me that sbt native builder can manage a lot of this stuff internally via sbt

**Comment by geoffjentry** [comment]
:+1:

File: Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ RUN mkdir /etc/service/cromwell && \
     apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D && \
     echo 'deb https://apt.dockerproject.org/repo ubuntu-trusty main' > /etc/apt/sources.list.d/docker.list && \
     apt-get update && \
-    apt-get install yq docker-engine=1.8.2-0~trusty && \
+    apt-get -yq install docker-engine=1.8.2-0~trusty && \
```
Comment (mcovarr) [inline Dockerfile]: Hmm

https://hub.docker.com/r/websend/app_env_drindex/~/dockerfile/

http://stackoverflow.com/questions/702248/how-can-i-automate-dpkg-apt-get

Comment (kshakir) [inline Dockerfile]: I'm assuming your Hmm is the equivalent a ToL without the out loud? 😉  Our Dockerfile may need other features to improve it, via other PRs, but at the moment the "install yq" is causing false negative build failures in jenkins.

Comment (mcovarr) [inline Dockerfile]: Sorry, I wasn't exactly clear.  😦 

What I tried to say too tersely was that there seem to be three different ways to attempt to express the standard automation options:

Our baseline Dockerfile:

`apt-get install yq ...`

This PR:

`apt-get -yq install ...`

The links above:

`apt-get install -yq ...`

The baseline variant seems incorrect, but it wasn't sure if the more common `apt-get install -yq` would be preferable to the changes here.

Comment (kshakir) [inline Dockerfile]: Gotcha, `apt-get install -yq` is more common. Still, the other lines use `apt-get -yq …`, and the `apt-get` [man page](http://manpages.ubuntu.com/manpages/xenial/en/man8/apt-get.8.html) lists the command as:

> SYNOPSIS
       apt-get [-asqdyfmubV] [-o=config_string] [-c=config_file]
               [-t=target_release] [-a=architecture] {update | upgrade |
               dselect-upgrade | dist-upgrade |
               install pkg [{=pkg_version_number | /target_release}]...  |
               remove pkg...  | purge pkg...  |
               source pkg [{=pkg_version_number | /target_release}]...  |
               build-dep pkg [{=pkg_version_number | /target_release}]...  |
               download pkg [{=pkg_version_number | /target_release}]...  |
               check | clean | autoclean | autoremove | {-v | --version} |
               {-h | --help}}

So I'm leaving it unless we get an override 2nd vote before this pr gets another thumbsup.

**Comment by scottfrazer** [comment]
👍

**Pull request merged**